packages/framework/src/Command/ReplaceObsoleteIconsCommand.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use Override;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+
+#[AsCommand(name: 'upgrade:replace-obsolete-icons', description: 'Replaces obsolete icons rendered with <i> tag by {{ ux_icons() }} function calls in Twig templates')]
+final class ReplaceObsoleteIconsCommand extends Command
+{
+    /**
+     * @param array<string, string> $iconAliases
+     */
+    public function __construct(
+        private readonly array $iconAliases,
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    #[Override]
+    protected function configure(): void
+    {
+        $this->addArgument('path', InputArgument::REQUIRED, 'The file or directory path containing Twig templates');
```

Comment:
We know the path so why are we asking for that?

Reply:
monorepo vs project usage? :slightly_smiling_face:

Reply:
I have added a default value in 0d101d8c3c